script/setup-dev

```diff
@@ -51,7 +51,7 @@ TMPDIR=$(mktemp -d -t node-tmp-XXXXXX)
   curl -sSLO ${DOWNLOAD_URL}
   set -x
   sha256sum --check "${DIR}/hashes/$(basename "${DOWNLOAD_URL}").sha256"
-  cat "${TMPFILE}" | sudo tar -xz -C /usr/local --strip-components=1
+  sudo tar -xz -C /usr/local --strip-components=1
```

Comment:
The old version looks like it failed to define `TMPFILE`, and this version is not specifying a file or a pipe as input to `tar`. Also, unrelated to this PR.

Reply:
oh yeah this was erroring for me so I was playing with it and forgot to remove